gulpfile.js

```diff
@@ -70,27 +73,31 @@ gulp.task('iconfont',
                 .pipe(gulp.dest('src/less/components'));
             }
         )
-        .pipe( gulp.dest('dist/css/fonts/') )
```

Comment:
I think we're still going to need to copy the fonts to the docs/kss/public/css/fonts folder for the live web version.  If we don't we'll have to have the server setup and configured to have the index of techne.yass.io pointing to the deep path and then navigating back in the folder structure could be problematice pending the server configuration